app/actions/GatewayActions.js

```diff
@@ -18,6 +18,57 @@ const onGatewayTimeout = (dispatch, gateway) => {
 };
 
 class GatewayActions {
+    fetchCoins({
```

Comment:
What's different to the other method?

Reply:
Which modal? Which other method?

Reply:
Please consider that there is no description for the codes, I cannot co-relate everything, I am trying to do it the standard way.

Reply:
In the same file there are two other "fetchCoins" methods. Difference?

Reply:
This file pull needs to be cancelled, I shouldn't have added this